search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -116,7 +123,13 @@
       ;;return all concepts if running with the system token
       concepts
       (let [acls (acl-helper/get-acls-applicable-to-token context)
-            applicable-field (-> concepts first :concept-type concept-type->applicable-field)
+            ;; drafts don't contain concept-type field, so we will need to derive it from concept-id
```

Comment:
I think we have some functions in the common-lib which do these now. Go to get-concept-type-of-draft
on the file common-lib/src/cmr/common/concepts.clj

Reply:
That function is for different purpose.  (get-concept-type-of-draft :collection-draft) returns :collection. 
What I need here, is given concept-id CD12345_PROV1, I need to return :collection-draft.